src/main/java/utils/DbUtils/Dao.java

```diff
@@ -10,35 +10,36 @@
 
 public class Dao {
 
+
         String dbUri = "jdbc:oracle:thin:@//florida.csr9s71rdbw2.us-east-1.rds.amazonaws.com:1521/FLORIDA";
         String userName = "florida";
         String password = "miamiflorida";
 
-        // We need to make these variables final so they cannot be changed by someone else
-        final String queryCreate = "INSERT INTO ADDRESS (ID, STREETNAME, CITY, ZIPCODE, STATE)" +
-                "VALUES(7, '311 True St', 'Richmond', '54544', 'VA')";
-        final String queryRead = "SELECT * FROM ADDRESS WHERE ZIPCODE=54544";
+        protected final String queryCreate = "INSERT INTO ADDRESS (ID, STREETNAME, CITY, ZIPCODE, STATE)" +
+                "VALUES(8, '311 False St', 'Austinos', '33333', 'TX')";
+        protected final String queryRead = "SELECT * FROM ADDRESS WHERE ZIPCODE=54544";
         final String queryUpdate = "UPDATE ADDRESS SET CITY = 'Miami', ZIPCODE='22222' WHERE ID='7'";
         final String queryDelete = "DELETE FROM ADDRESS WHERE ID=7";
 
-        // let us extract this and make a method
-        Connection conn = DriverManager.getConnection(dbUri, userName, password);
-        Statement statement = conn.prepareStatement(queryCreate);
+        Connection conn = null;
+        Statement statement = null;
+        ResultSet rs = null;
+
+           protected void executeQuery() throws SQLException {
+            conn = DriverManager.getConnection(dbUri, userName, password);
```

Comment:
extract getConnection method from executeQuery Method
(hint: getConnection method will have return type as Connection)

Write seperate executeQuery methods for each CRUD operation
and test them seperately